app/views/widgets/lists.html

```diff
@@ -10,19 +10,15 @@
         {% elif status == 'complete' %}
             <!-- Check your answers -->
             {% if alreadyDeclared === 'false' %}
-                <p><a href="{{checkYourAnswersURL}}">{{content.taskComplete}}</a></p>
+                <p class="task-item-content"><a href="{{checkYourAnswersURL}}">{{content.checkYourAnswers}}</a></p>
```

Comment:
As we have the `.task-item-content` as a wrapper `<div>` on other pages, I think we should so the same here for consistency and as before move it out of the `if` statement.